CrewChiefEx/CrewChiefEx.py

```diff
@@ -128,23 +138,46 @@ def acMain(ac_version):
 
 def acUpdate(deltaT):
     global siminfo
-    global l_lapcount, lapcount,l_driver,l_drivers,maxSlotId,l_flag
-    updateSharedMemory()
+    global l_lapcount, lapcount,l_driver,l_drivers,maxSlotId,l_flag, timer1
+    
+    timer1 += deltaT
+    
+    if timer1 > 0.050:
```

Comment:
I will comment out all the "debug"  below the 'if' timer, its really only for developing and checking values, if that ok